django_tables2/columns/templatecolumn.py

```diff
@@ -56,11 +67,18 @@ def render(self, record, table, value, bound_column, **kwargs):
         additional_context = {
             "default": bound_column.default,
             "column": bound_column,
-            "record": record,
+            self.context_object_name: record,
             "value": value,
             "row_counter": kwargs["bound_row"].row_counter,
         }
-        additional_context.update(self.extra_context)
+
+        extra_context = self.extra_context
+        if callable(extra_context):
```

Comment:
ik had uit de docstring begrepen dat extra_context een dict was - dus dan zou ik de callable meegeven in een dict ... maar blijkbaar moet je een callable toewijzen aan extra_context? Dus voor mij is de docstring van verwarrend

Reply:
You should return a callable returning a dict.